app/storage/postgres/user.go

```diff
@@ -346,3 +345,22 @@ func (s *UserStorage) GetByAPIKey(apiKey string) (*models.User, error) {
 	}
 	return user, nil
 }
+
+func (s *UserStorage) FindLike(query string) ([]*models.User, error) {
+	var users []*dbUser
+	err := s.trx.Select(&users, `
+		SELECT id, name, email, tenant_id, role, status 
+		FROM users 
+		WHERE name like $1 || '%'`, query)
```

Comment:
Two other filters we are missing here:
- tenant_id, so that it doesn't list users from another tenant.
- status, so that it doesn't not list deleted users.

Have a look on `GetAll()` function, it does a filter on both fields that we also need to filter here.